src/xmlnode.h

```diff
@@ -246,6 +246,28 @@ class XmlNode
      */
     bool isText() const;
 
+    /**
+     * @brief Check if this @c XmlNode is a text node representing a basic number.
+     *
+     * Checks whether this @c XmlNode is a text node representing a basic number.
+     * Returns @ true if so, and @c false otherwise.
+     *
+     * @return @c true if this @c XmlNode is a text node representing a basic number
+     * and @c false otherwise.
+     */
+    bool isBasicNumber() const;
```

Comment:
This function has to be called `isBasicReal`, it is not testing if the node is a number.

Reply:
3170dc65